traceKeyExtractor.py

```diff
@@ -116,9 +83,9 @@ def discoverKeys():
         #Run PM3 with the trace
         # -o means run without connecting to PM3 hardware
         # -c specifies commands within proxmark 3 software
-        cmd_list = [pm3Command,"-o","-c", f"trace load -f {trace}; trace list -1 -t mf -f {dictionaryFilepath}; exit"];
+        cmd_list = [pm3Location / pm3Command,"-o","-c", f"trace load -f {trace}; trace list -1 -t mf -f {dictionaryFilepath}; exit"];
         print(f"Viewing tracelog with {len(keyList)} discovered keys")
-        print(" ".join(cmd_list))
+        print(f"pm3 {" ".join(cmd_list[1:])}")
```

Comment:
```suggestion
        print(f"pm3 {' '.join(cmd_list[1:])}")
```
Fixes syntax error when double quotes are used inside double quotes.

Reply:
In Python 3.12, this isn't a syntax error and works as intended, but since the script requires 3.6 or higher, I'll make the appropriate changes!